In [8]:
import pandas as pd

pesticide = pd.read_csv('2014.csv')
pesticide.head()

COMPOUND  YEAR  STATE_CODE  COUNTY_CODE  LOW_ESTIMATE  HIGH_ESTIMATE
0    2,4-D  2014           1            1        1698.6         1885.5
1    2,4-D  2014           1            3        7513.6         8472.4
2    2,4-D  2014           1            5        2613.6         2889.4
3    2,4-D  2014           1            7        1259.2         1277.7
4    2,4-D  2014           1            9        7590.5         7756.1

In [12]:
pesticide.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 392433 entries, 0 to 392432
Data columns (total 6 columns):
 #   Column         Non-Null Count   Dtype  
---  ------         --------------   -----  
 0   COMPOUND       392433 non-null  object 
 1   YEAR           392433 non-null  int64  
 2   STATE_CODE     392433 non-null  int64  
 3   COUNTY_CODE    392433 non-null  int64  
 4   LOW_ESTIMATE   273406 non-null  float64
 5   HIGH_ESTIMATE  392433 non-null  float64
dtypes: float64(2), int64(3), object(1)
memory usage: 18.0+ MB


In [11]:
pesticide.describe(include = 'all')

COMPOUND      YEAR     STATE_CODE    COUNTY_CODE  LOW_ESTIMATE  \
count       392433  392433.0  392433.000000  392433.000000  2.734060e+05   
unique         388       NaN            NaN            NaN           NaN   
top     Glyphosate       NaN            NaN            NaN           NaN   
freq          3062       NaN            NaN            NaN           NaN   
mean           NaN    2014.0      30.725670      94.597531  1.534195e+03   
std            NaN       0.0      14.774077      85.292216  2.099562e+04   
min            NaN    2014.0       1.000000       1.000000  0.000000e+00   
25%            NaN    2014.0      19.000000      35.000000  1.000000e+00   
50%            NaN    2014.0      29.000000      77.000000  1.250000e+01   
75%            NaN    2014.0      45.000000     129.000000  1.343000e+02   
max            NaN    2014.0      56.000000     810.000000  5.507146e+06   

        HIGH_ESTIMATE  
count    3.924330e+05  
unique            NaN  
top               NaN  
freq              NaN  
mean     1.224855e+03  
std      1.777421e+04  
min      0.000000e+00  
25%      1.500000e+00  
50%      1.650000e+01  
75%      1.416000e+02  
max      5.507146e+06

The most frequent used pesticide in the United States is Glyphosate, which is used in 3062 measurements out of 392433 measurements.